uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/AbsenceTypeTest.php

```diff
@@ -718,21 +714,18 @@ public function testNonExpiredToilRequestsAreDeletedWhenToilIsDisabledForAbsence
       'type_id' => $absenceType->id,
       'contact_id' => 1,
       'status_id' => 1,
```

Comment:
Oh, I just realized the TOILRequestFabricator also has a default value for the status_id, so you can remove it as well